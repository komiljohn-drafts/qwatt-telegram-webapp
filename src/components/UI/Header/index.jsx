import { Container } from "@mui/material";
export default function Header() {
  return (
    <header>
      <Container />
    </header>
  );
}
