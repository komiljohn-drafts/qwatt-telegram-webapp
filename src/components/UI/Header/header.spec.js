import Header from "./index";
import { RouterContext } from "next/dist/shared/lib/router-context";
import { createMockRouter } from "@/test-config/createMockRouter";
import { render } from "@testing-library/react";

describe("About component", () => {
  it("should render Footer component", () => {
    const { asFragment } = render(
      <RouterContext.Provider value={createMockRouter()}>
        <Header />
      </RouterContext.Provider>
    );
    expect(
      asFragment(
        <RouterContext.Provider value={createMockRouter()}>
          <Header />
        </RouterContext.Provider>
      )
    ).toMatchSnapshot();
  });
});
