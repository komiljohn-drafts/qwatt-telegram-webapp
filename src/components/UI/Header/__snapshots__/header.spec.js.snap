// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component should render Footer component 1`] = `
<DocumentFragment>
  <header
    class="header"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
    >
      <div
        class="box"
      >
        <a
          class="logo"
          href="/"
        >
          <h2>
            Logo
          </h2>
        </a>
        <nav>
          <ul>
            <li>
              <a
                href="/"
              >
                home
              </a>
            </li>
            <li>
              <a
                href="/blog"
              >
                blog
              </a>
            </li>
            <li>
              <a
                href="/about"
              >
                about
              </a>
            </li>
          </ul>
        </nav>
        <div
          class="langs"
        >
          <ul>
            <li>
              <a
                href="/"
              >
                ru
              </a>
            </li>
            <li>
              <a
                href="/"
              >
                uz
              </a>
            </li>
            <li>
              <a
                href="/"
              >
                en
              </a>
            </li>
          </ul>
        </div>
      </div>
    </div>
  </header>
</DocumentFragment>
`;
